contrib/zstd/README.md

```diff
@@ -0,0 +1,20 @@
+﻿
+# ZSTD Contrib Library Manual
+
+
+## Introduction
+
+ZSTD on ZFS uses a heavily cut down, but otherwise unmodified version of ZSTD.
+This `contrib` contains the  unmodified ZSTD library, please do not edit these files.
+
+## Updating ZSTD
+
+To update ZSTD the following steps need to be taken:
+
+1.  Download the latest archive of ZSTD
+2. Replace (not merge): `common`, `compress`, `decompress` and `zstd.h` with the new version from the `lib` folder in the downloaded archive of zstd.
+3. Make sure any newly requered files and/or folders are also included.
```

Comment:
`requered` -> `required`

Reply:
Thanks.
@c0d3z3r0 Mind a quick typo fix after codecov report gets in?

Reply:
@c0d3z3r0 ?

Reply:
oops... working on it

Reply:
@rlaager @Ornias1993 I reworked this a bit. Can you check it again, please?

Reply:
@c0d3z3r0 No problem, It was also a bit my fault...
I thought about sending you a PR, but asked you to do it and forgot to check in on it.

As it's 00:48 here now, I'll look into it somewhere this week.... Family is expecting a not-zombie tomorrow. ;)

Reply:
Looks good other than "requiered" still needs typo fixing.

Reply:
@rlaager wtf... brain.exe crashed ;)